core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -486,7 +481,7 @@ private[spark] class Executor(
    * Download any missing dependencies if we receive a new set of files and JARs from the
    * SparkContext. Also adds any new JARs we fetched to the class loader.
    */
-  private def updateDependencies(newFiles: HashMap[String, Long], newJars: HashMap[String, Long]) {
+  private def updateDependencies(newFiles: Map[String, Long], newJars: Map[String, Long]) {
```

Comment:
@wzhfy 
The source of this parameter is `sc.addedFiles` and `sc.addedJars`, Their types are `mutable.Map[String, Long]` , This change is reasonable.